exercises/python-1-b/python_1_b.py

```diff
@@ -40,7 +40,7 @@
     "6": "SIX",
     "7": "SEVEN",
     "8": "EIGHT",
-    "9": "NINER",
+    "9": "NINE",
```

Comment:
Not a typographical error.  It is indeed "NINER".

Reply:
Reasons:  ATC (air traffic control), military communications, and international communication where radio noise can be significant, it is important to not confuse words. I had to scroll up and make sure that "oscar" and/or "mike" where used.  Technically speaking, we were trained to say "fife" instead of the softer sounding "v" in "five" and similar clarifying sounds where communication mistakes can cost lives.

The NATO alphabet including numbers can be found [here](https://en.wikipedia.org/wiki/NATO_phonetic_alphabet) adopted by others, including the ICAO (International Civil Aviation Organization).

Reply:
Right, I did look at the WIkipedia article but only looked at the "Telephony"/"Code Word" columns.
Thanks for the explanation!

Reply:
Can you close PR if you think it is good as is.